
Formerly part of 01-time-slider.patch, header changes at 1.26.0
required an updated patch.

diff -ur caja-1.26.0.orig/src/caja-window.h caja-1.26.0/src/caja-window.h
--- caja-1.26.0.orig/src/caja-window.h	2021-08-04 08:10:30.000000000 +0000
+++ caja-1.26.0/src/caja-window.h	2021-11-26 17:39:50.560575043 +0000
@@ -160,6 +160,8 @@
         gboolean           new_tab);
 void             caja_window_allow_reload         (CajaWindow    *window,
         gboolean           allow);
+void             caja_window_allow_restore        (CajaWindow    *window,
+        gboolean           allow);
 void             caja_window_allow_up             (CajaWindow    *window,
         gboolean           allow);
 void             caja_window_allow_stop           (CajaWindow    *window,
